README.md

```diff
@@ -24,8 +24,24 @@ There are no dependencies.
 
 ## Usage
 
-Below is a sample playbook:
+Below are sample usages for playbook:
 
+##### For deploying agents to Non-Multi-Tenancy Deep Security Manager
```

Comment:
For deploying [...] -> For deploying agents to a non-multi-tenant Deep Security Manager